timer/tablestore/store.go

```diff
@@ -114,6 +119,11 @@ func (s *tableTimerStoreCore) List(ctx context.Context, cond api.Cond) ([]*api.T
 	}
 	defer back()
 
+	if sessVars := sctx.GetSessionVars(); sessVars.GetEnableIndexMerge() {
```

Comment:
Is there any plan we could remove this in the future? IMO it's quite confusing to the others.

Reply:
It's a limitation of multi-value index that is documented here: https://docs.pingcap.com/tidb/dev/choose-index#use-a-multi-valued-index . I think I can add some comments

Reply:
Got it, please add a comment about this.